lisa/features/infiniband.py

```diff
@@ -1,13 +1,37 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
 
+import re
+from typing import List, Match, Tuple
 
 from lisa.feature import Feature
+from lisa.util import LisaException
 
 FEATURE_NAME_INFINIBAND = "Infiniband"
 
 
 class Infiniband(Feature):
+    _ib_info_pattern = re.compile(
```

Comment:
add an example above.

Reply:
I will add example output, but now I'm thinking I could probably better implement this with something like
`(\s*(?<id>\S*):\s*(?<value>.*)\n?)` multiline
That way I can easily create a dictionary of the values instead of naming every match group.

Reply:
yes, you can capture name/value pairs. try `find_groups_in_lines` in `util/__init__.py`